ci/images/spring-credhub-ci/Dockerfile

```diff
@@ -45,6 +45,12 @@ RUN curl -Lo ytt "https://github.com/vmware-tanzu/carvel-ytt/releases/download/v
 RUN curl -L "https://github.com/docker/compose/releases/download/1.29.2/docker-compose-$(uname -s)-$(uname -m)" \
     -o /usr/local/bin/docker-compose && chmod +x /usr/local/bin/docker-compose
 
+RUN \
```

Comment:
Could we add a comment for future reference, for instance to the upstream issue https://github.com/docker/cli/issues/4807?

Reply:
Good idea

Reply:
Done, thanks!